modules/visual_script/visual_script.cpp

```diff
@@ -674,30 +582,25 @@ void VisualScript::rename_variable(const StringName &p_name, const StringName &p
 
 	ERR_FAIL_COND(!String(p_new_name).is_valid_identifier());
 
-	ERR_FAIL_COND(functions.has(p_new_name));
+	ERR_FAIL_COND(functions.has(p_new_name)); //why?
```

Comment:
Good question, which needs to be resolved :)

Reply:
When editing a member, visual_script_editor.cpp _member_edited checks to make sure that the name is not already a function. variable, or signal. I assume that this reassuring that this still remains the case.